### PR TITLE
Add content/pt-br/docs/reference/glossary/cla.md

### PR DESCRIPTION
**Pull request by MrErlison**
Added translate content/pt-br/docs/reference/glossary/cla.md

/assign @edsoncelio

### CODE REVIEW

File: content/pt-br/docs/reference/glossary/cla.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+aka: 
+tags:
+- community
+---
+ Termos sob os quais um {{< glossary_tooltip text="colaborador" term_id="contributor" >}} concede uma licença a um projeto de código aberto por suas contribuições.
+
+<!--more--> 
+
+Os CLAs ajudam a resolver disputas legais envolvendo material contribuído e propriedade intelectual (do inglês - intellectual property IP).
```
Comment (sftim) [inline content/pt-br/docs/reference/glossary/cla.md:17]:
```suggestion
Os CLAs ajudam a resolver disputas legais envolvendo material contribuído e propriedade intelectual.
```
No need to mention IP.

Comment (MrErlison) [reply content/pt-br/docs/reference/glossary/cla.md]: You're right.

**Comment by rikatz** [comment]
/label tide/merge-method-squash
/lgtm
/approve
Thanks

**Pull request merged**